README.md

```diff
@@ -63,6 +66,38 @@ $ oc adm taint nodes <NodeNames> node.ocs.openshift.io/storage=true:NoSchedule
 
 The OCS operator can be installed into an OpenShift cluster using Operator Lifecycle Manager (OLM).
 
+#### Option 1: Using `make install`
+If you have a development environment or private image and want to install the OCS operator, follow the steps below:
+
+1.Label Worker Nodes:  
+OCS Operator will install its components only on nodes labeled for OCS.
+```console
+oc label nodes <NodeName1> cluster.ocs.openshift.io/openshift-storage=''
+oc label nodes <NodeName2> cluster.ocs.openshift.io/openshift-storage=''
+oc label nodes <NodeName3> cluster.ocs.openshift.io/openshift-storage=''
+```
+
+2.Set Environment Variables:  
+Define the required variables for your private image:
+```console
+export REGISTRY_NAMESPACE=<your-registry-namespace>
+export IMAGE_TAG=<your-image-tag>
+```
+
+3.Run the following command:
+
+```console
+make install
+```
+
+4.Verify Installation:  
+Once the make install process completes, verify the status of the ClusterServiceVersion (CSV):
+```console
+oc get csv -n openshift-storage
+```
+
+#### Option 2: Using Pre-Built YAML
+To label nodes, run:
 For quick install using pre-built container images, deploy the [deploy-olm.yaml](deploy/deploy-with-olm.yaml) manifest.
```

Comment:
@iamniting Does the deploy with old yaml work anymore after all the changes that have happened?

Reply:
If not probabaly we can get rid of this

Reply:
It should work as it creates all required subscription, ocs, rook, noobaa.